raw.go

```diff
@@ -95,6 +95,12 @@ func (c *Conn) SetBPF(filter []bpf.RawInstruction) error {
 	return c.p.SetBPF(filter)
 }
 
+// SetPromisc enables/disables the interface promiscuous mode
+// 0 == disable, 1 == enable
+func (c *Conn) SetPromisc(m int) error {
```

Comment:
Let's go with the full name: `SetPromiscuous` and a `enable bool` parameter since it appears 0/1 are the only valid states.